configs/common/packages.yaml

```diff
@@ -9,290 +9,255 @@
         glu: [openglu]
         jpeg: [libjpeg-turbo]
         lapack: [openblas]
+        pkgconfig: [pkg-config]
         yacc: [bison]
         zlib-api: [zlib-ng]
     #
     awscli-v2:
-      require: "~examples"
+      require: '~examples'
     bacio:
-      version: ['2.4.1']
+      require: '@2.4.1'
     bison:
-      version: ['3.8.2']
-    # 1.85 incompatible with ecflow@5.11.4 - use latest "good version"
+      require: '@3.8.2'
+    # 1.85 incompatible with ecflow@5.11.4 - use latest 'good version'
     boost:
-      require: "@1.84 ~atomic +chrono +date_time +exception +filesystem ~graph ~iostreams ~locale ~log ~math ~mpi ~numpy +pic +program_options +python ~random +regex +serialization ~signals +system +test +thread +timer ~wave cxxstd=17 visibility=hidden"
+      require: '@1.84 ~atomic +chrono +date_time +exception +filesystem ~graph ~iostreams ~locale ~log ~math ~mpi ~numpy +pic +program_options +python ~random +regex +serialization ~signals +system +test +thread +timer ~wave cxxstd=17 visibility=hidden'
     bufr:
-      version: ['12.0.1']
-      variants: +python
+      require: '@12.0.1 +python'
     cairo:
-      variants: +pic
+      require: '+pic'
     cdo:
-      version: ['2.2.0']
-      variants: ~openmp
+      require: '@2.2.0 ~openmp'
     cmake:
       version: ['3.27.9']
-      variants: +ownlibs
+      require: '+ownlibs'
     # Attention - when updating also check the various jcsda-emc-bundles env packages
     crtm:
-      version: ['2.4.0.1']
-      variants: +fix
+      require: '+fix'
     ecbuild:
-      version: ['3.7.2']
+      require: '@3.7.2'
     eccodes:
-      version: ['2.33.0']
-      variants: +png
+      require: '@2.33.0 +png'
     ecflow:
-      version: ['5.11.4']
-      variants: +ui
+      require: '@5.11.4 +ui'
```

Comment:
Does this work with external ecflow packages?

Reply:
It needs to be fully tested, but yes-- the idea is that each site config will use `ecflow::`to override.